tests/testthat/test-count.R

```diff
@@ -852,3 +852,21 @@ test_that("nested count layers error out when you try to add a total row", {
 
     expect_snapshot_error(build(tab))
 })
+
+test_that("Tables with pop_data can accept a layer level where", {
+
+  load(test_path('adsl.Rdata'))
+  load(test_path('adae.Rdata'))
+  t <- tplyr_table(adae, TRTA) %>%
+    set_pop_data(adsl) %>%
+    set_pop_treat_var(TRT01A) %>%
+    set_pop_where(TRUE) %>%
+    add_layer(
+      group_count(AEDECOD, where = AEREL != "NONE") %>%
+        set_distinct_by(USUBJID) %>%
+        set_denom_where(TRUE)
```

Comment:
@mstackhouse I updated the test here to remove the set_denom_where. I had that in originally because it caused some issues earlier when I was developing the bug fix. You're right it doesn't do anything in this case because it will be overwritten anyway.

I've updated the test to include n, distinct_n, pct, and distinct_pct and use dput to get the whole output.